docs/user-guide/slicing.ipynb

```diff
@@ -6,10 +6,19 @@
    "source": [
     "# Slicing\n",
     "\n",
-    "Data in a [Variable](../generated/scipp.Variable.rst#scipp.Variable) or [Dataset](../generated/scipp.Dataset.rst#scipp.Dataset) can be indexed in a similar manner to NumPy and xarray.\n",
+    "Objects in scipp can be sliced in two ways. The most common is by by index based slicing. A second, more limited approach is to use value based slicing based on coordinate values."
```

Comment:
Is it worth adopting terminology closer to xarray's? See http://xarray.pydata.org/en/stable/indexing.html.